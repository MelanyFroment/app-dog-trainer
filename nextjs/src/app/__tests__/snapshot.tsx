import React from "react";
import { render } from "@testing-library/react";
import Page from "../page";

// Mock du router Next.js
jest.mock("next/navigation", () => ({
  useRouter: () => ({
    push: jest.fn(),
    replace: jest.fn(),
    prefetch: jest.fn(),
  }),
}));

describe("Snapshot homepage", () => {
  it("renders homepage unchanged", () => {
    const { container } = render(<Page />);
    expect(container).toMatchSnapshot();
  });
});
