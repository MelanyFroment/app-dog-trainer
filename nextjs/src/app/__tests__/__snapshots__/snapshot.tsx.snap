// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot homepage renders homepage unchanged 1`] = `
<div>
  <div
    class="p-4"
  >
    <h1
      class="text-2xl font-bold mb-4"
    >
      Home
    </h1>
    <div
      class="flex gap-4"
    >
      <a
        class="text-blue-500 hover:underline"
        href="/login"
      >
        Se connecter
      </a>
      <a
        class="text-blue-500 hover:underline"
        href="/about"
      >
        About
      </a>
      <button
        class="text-red-500 hover:underline"
      >
        Se d√©connecter
      </button>
    </div>
  </div>
</div>
`;
